### DISCUSSION TITLE
problems with dota

### DISCUSSION THREAD

**Discussion by AlexeyProg** [AveYo/ShowMMR#18] [Q&A]
Hello! I made a mistake, i think. And after entering in dota i dont see main menu. There are glares and a blurry screen. Then i decided to reinstall the game but nothing changed. What can you advice to do with that ? 
I think that i incorrect swap files in local dirs

**Comment by AveYo** [comment]
Probably gpu driver issue / bad shader cache?

Assuming you've sorted out your drivers and os settings, what you can do in Dota is:

1st,
add just launch options: `-dx11 -autoconfig_level 0 +reset_gameconvars +r_dota_bloom_compute_shader 0` 
open dota once, then remove the launch options as they did their job  

2nd, 
delete -your-Steam-folder-/steamapps/shadercache/570 
delete -your-Steam-folder-/steamapps/common/dota 2 beta/game/dota/shadercache  
delete ../game/dota/gameinfo.gi
delete ../game/dota/gameinfo_branchspecific.gi
delete / rename any client-side mods folders (game/dota_mods or etc)
there should not be any other folder in ../game/ except
dota\
bin\
dota_lv\
core\
dota_schinese\
dota_russian\
dota_koreana\
dota_addons\


3rd,
Steam Library - Dota - Properties - Installed files - **Verify integrity of game files**
